titles/admin-rhdh/title-admin.adoc

```diff
@@ -10,11 +10,22 @@ The {product} is an enterprise-grade, open developer platform that you can use t
 //customer support links
 include::artifacts/snip-customer-support-info.adoc[]
 
-// install operator
-include::modules/admin/proc-install-operator.adoc[leveloffset=+1]
+== Installing {product-short}
+
+You can install {product} on the following platforms:
+
+* {ocp-brand-name} ({ocp-very-short})
+* {eks-brand-name} ({eks-short})
+* {aks-brand-name} ({aks-short})
```

Comment:
In the future, we can consolidate some of this information into a single file while eliminating some redundancy in other modules. 

So in addition to telling the user that they can install RHDH on each of these platforms, we can go ahead also tell them the following information: 

- You can use either the Operator or Helm chart to install RHDH on each of these platforms 
- You can use the console or the CLI

But I would wait until we can replace the info in assemby-install- rhdh-ocp.adoc with more conceptual info about when / why a user would want to install RHDH on one platform over another, so it isn't completely eliminated or left blank.